packages/ui/react/src/components/Forms/Input/Input.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+
+import { ErrorMessage } from '../../Primitives/ErrorMessage'
+import { Container, InputText } from './Input.styled'
+import { Label } from 'src/components/Forms/Label/Label'
+import type { LabelProps } from 'src/components/Forms/Label/Label'
+
+type InputProps = React.InputHTMLAttributes<HTMLInputElement> & {
+  /** React ref for direct DOM manipulation */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ref?: any
+  /** HTML input type (text, number, email, etc) */
+  type: React.HTMLInputTypeAttribute
+  /** Removes increment/decrement arrows from number inputs */
+  removeArrows?: boolean
+  /** Error message configuration */
+  error?: React.ComponentPropsWithoutRef<typeof ErrorMessage>['error']
+  /** Alternative ref prop for input element */
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  inputref?: any
+  /** Label component props */
+  label?: LabelProps
```

Comment:
Can you apply the suggestion below to use consistent prop naming? As we already have inputProps

```suggestion
  /** Props passed directly to Label component */
  labelProps?: LabelProps
```